src/public/less/styles.less

```diff
@@ -56,11 +56,11 @@ body {
 	background-color: #000;
 }
 
-body, div, p, span, li, blockquote, small {
+body, div, p, ul, blockquote {
```

Comment:
only want these on block level elements. things like `span` should inherit from their parent.